packages/table/src/cell/editableCell.tsx

```diff
@@ -53,12 +58,20 @@ export interface IEditableCellState {
     dirtyValue?: string;
 }
 
+@HotkeysTarget
 export class EditableCell extends React.Component<IEditableCellProps, IEditableCellState> {
     public static defaultProps = {
         truncated: true,
         wrapText: false,
     };
 
+    private cellRef: HTMLElement;
+    private refHandlers = {
+        cell: (ref: HTMLElement) => {
+            this.cellRef = ref;
+        },
```

Comment:
nit: Change this to a one-liner:

```tsx
cell: (ref: HTMLElement) => this.cellRef = ref,
```

Reply:
Prettier does this. Or at least, adds parens if I don't use curlies, which is honestly silly.